include/FreeRTOS.h

```diff
@@ -460,6 +460,10 @@ hold explicit before calling the code. */
 	#define traceQUEUE_SEND( pxQueue )
 #endif
 
+#ifndef traceQUEUE_SET_SEND

```

Comment:
To ensure that the change is backward compatible, would you please change it to the following:

```c
#ifndef traceQUEUE_SET_SEND
	#define traceQUEUE_SET_SEND traceQUEUE_SEND
#endif

#ifndef traceQUEUE_SEND
	#define traceQUEUE_SEND( pxQueue )
#endif
```

Thanks.

Reply:
Done.